### PR TITLE
participant-state: Remove the aliases to `Ref.LedgerString`.

### PR DESCRIPTION
**Pull request by SamirTalwar-DA**
Use the aliases directly in `Ref` instead.

### Pull Request Checklist

- [x] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/main/CONTRIBUTING.md)
- [x] Include appropriate tests
- [x] Set a descriptive title and thorough description
- [x] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [ ] Include changelog additions in one or more commit message bodies between the `CHANGELOG_BEGIN` and `CHANGELOG_END` tags
- [ ] Normal production system change, include purpose of change in description

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

File: ledger/participant-state/src/main/scala/com/daml/ledger/participant/state/v2/package.scala [hunk 1]
```diff
@@ -45,26 +43,4 @@ import com.daml.lf.data.Ref
...
-  /** Identifiers for submitting client applications. */
-  val ApplicationId: Ref.LedgerString.type = Ref.LedgerString
-  type ApplicationId = Ref.LedgerString
-
-  /** Identifiers used to correlate submissions with results. */
-  val SubmissionId: Ref.LedgerString.type = Ref.LedgerString
-  type SubmissionId = Ref.LedgerString
-
-}
+package object v2
```
Comment (fabiotudone-da) [inline ledger/participant-state/src/main/scala/com/daml/ledger/participant/state/v2/package.scala:46]: Do we need to keep this file at all?

File: ledger/participant-state/src/main/scala/com/daml/ledger/participant/state/v1/package.scala [hunk 2]
```diff
@@ -52,26 +50,4 @@ import com.daml.lf.data.Ref
...
-  /** Identifiers for submitting client applications. */
-  val ApplicationId: Ref.LedgerString.type = Ref.LedgerString
-  type ApplicationId = Ref.LedgerString
-
-  /** Identifiers used to correlate admin submission with results. */
-  val SubmissionId: Ref.LedgerString.type = Ref.LedgerString
-  type SubmissionId = Ref.LedgerString
-
-}
+package object v1
```
Comment (fabiotudone-da) [inline ledger/participant-state/src/main/scala/com/daml/ledger/participant/state/v1/package.scala:53]: Do we need to keep this file at all?

File: daml-lf/data/src/main/scala/com/digitalasset/daml/lf/data/Ref.scala [hunk 3]
```diff
@@ -49,6 +49,10 @@ object Ref {
   type CommandId = LedgerString
   val CommandId: LedgerString.type = LedgerString
 
+  /** Identifiers used to correlate an admin submission with results. */
```
Comment (fabiotudone-da) [inline daml-lf/data/src/main/scala/com/digitalasset/daml/lf/data/Ref.scala:52]: Possibly an ignorant question but why an "admin submission|?

File: ledger/participant-state/src/main/scala/com/daml/ledger/participant/state/v2/package.scala (hunk 1 above)
Comment (nicu-da) [inline ledger/participant-state/src/main/scala/com/daml/ledger/participant/state/v2/package.scala:46]: does this still make sense to have it empty?

**Review by nicu-da — APPROVED** [review]

File: daml-lf/data/src/main/scala/com/digitalasset/daml/lf/data/Ref.scala (hunk 3 above)
Comment (fabiotudone-da) [reply daml-lf/data/src/main/scala/com/digitalasset/daml/lf/data/Ref.scala:52]: Why only for admin ones though?

**Review by miklos-da — APPROVED** [review]

File: ledger/participant-state/src/main/scala/com/daml/ledger/participant/state/v2/package.scala (hunk 1 above)
Comment (SamirTalwar-DA) [reply ledger/participant-state/src/main/scala/com/daml/ledger/participant/state/v2/package.scala:46]: I just kept it for the commentary. I think it's more useful as Scaladoc than as a README.

File: ledger/participant-state/src/main/scala/com/daml/ledger/participant/state/v1/package.scala (hunk 2 above)
Comment (SamirTalwar-DA) [reply ledger/participant-state/src/main/scala/com/daml/ledger/participant/state/v1/package.scala:53]: See above.

File: daml-lf/data/src/main/scala/com/digitalasset/daml/lf/data/Ref.scala (hunk 3 above)
Comment (SamirTalwar-DA) [reply daml-lf/data/src/main/scala/com/digitalasset/daml/lf/data/Ref.scala:52]: I think the submission ID was historically only used for the admin interface. However, on closer inspection, we use it everywhere. So I should probably change that.

File: ledger/participant-state/src/main/scala/com/daml/ledger/participant/state/v2/package.scala (hunk 1 above)
Comment (SamirTalwar-DA) [reply ledger/participant-state/src/main/scala/com/daml/ledger/participant/state/v2/package.scala:46]: See Fabio's comment.

File: daml-lf/data/src/main/scala/com/digitalasset/daml/lf/data/Ref.scala (hunk 3 above)
Comment (miklos-da) [reply daml-lf/data/src/main/scala/com/digitalasset/daml/lf/data/Ref.scala]: Indeed, now we have submission ID on all `WriteService` calls (for transactions it used to only have command ID in v1).

Comment (SamirTalwar-DA) [reply daml-lf/data/src/main/scala/com/digitalasset/daml/lf/data/Ref.scala]: I've changed the comment.

File: daml-lf/data/src/main/scala/com/digitalasset/daml/lf/data/Ref.scala [hunk 4]
```diff
@@ -45,15 +45,19 @@ object Ref {
   type ApplicationId = LedgerString
   val ApplicationId: LedgerString.type = LedgerString
 
-  /** Identifiers used to correlate submission with results. */
+  /** Identifiers used to correlate a command submission with ts result. */
```
Comment (miklos-da) [inline daml-lf/data/src/main/scala/com/digitalasset/daml/lf/data/Ref.scala:48]:
```suggestion
  /** Identifiers used to correlate a command submission with its result. */
```

**Review by fabiotudone-da — APPROVED** [review]

**Pull request merged**